low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -42,6 +42,9 @@ def __init__(self, args: list):
         self.name = "sspl_test"
         self.plan = "sanity"
         self.coverage_enabled = self.args.coverage
+        self.test_dir = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test"
+        if self.args.v2:
+            self.test_dir = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test/functional_tests"
```

Comment:
It can only help if we plan to keep current test framework. Once we move to v2 and delete existing code, this versioning of folder will become obsolete.

Reply:
No we can maintain both, we will just move to using v2

Reply:
Done.